pkg/handlers/app.go

```diff
@@ -20,6 +20,51 @@ import (
 	"github.com/replicatedhq/kots/pkg/version"
 )
 
+func (h *Handler) GetPendingApps(w http.ResponseWriter, r *http.Request) {
+	sess := session.ContextGetSession(r)
```

Comment:
this is not needed, it's already handled in the session middleware (should be removed from `ListApps` handler as well)